src/LooCompare.jl

```diff
@@ -0,0 +1,134 @@
+import Base.show
+
+
+"""
+
+# LooCompare
+
+A struct containing the results of PsisLoo model comparisom.
+
+$(FIELDS)
+
+# Extended help
+
+# Fields
+  - `psis::Vector{PsisLoo}`                      : A vector of PsisLoo objects.
+  - `table::KeyedArray`                          : Comparison table.
+
+"""
+struct LooCompare
+    psis::Vector{PsisLoo}
+    table::KeyedArray
+end
+
+"""
+
+# loo_compare
+
+Construct a PsisLoo comparison table for loglikelihood matrices.
+
+$(SIGNATURES)
+
+# Extended help
+
+### Required arguments
+```julia
+* `loglikelihoods::Vector{Array{Float64, 3}}`    : Vector of loglikelihood matrices
+```
+
+### Optional arguments
+```julia
+* `model_names=nothing`                          : Optional specify models
+* `sort_models=true`                             : Sort models
+```
+
+### Return values
+```julia
+* `result::LooCompare`                           : LooCompare object
+```
+
+"""
+function loo_compare(
+    loglikelihoods::Vector{Array{Float64, 3}};
+    model_names=Union{Nothing, Vector{AbstractString}}, 
```

Comment:
Is typing necessary here? Would it be sufficient to use` model_names = nothing` ?

Reply:
Probably, in fact that's what I had early on. Dropped typing constraint.